### PR TITLE
Update contributor instructions

### PR DESCRIPTION
**Pull request by dholbach**
We don't use LP anymore, update instructions.

### CODE REVIEW

**Comment by zyga** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -79,15 +79,17 @@ to have it available in `$GOPATH/bin`
...
 Contributions are always welcome! Please make sure that you sign the
 Canonical contributor licence agreement at
-http://www.ubuntu.com/legal/contributors 
+http://www.ubuntu.com/legal/contributors
 
 To get the source and propose a merge, this is what typically needs to
 be done:
 
-     bzr branch lp:snappy my-work
+     git clone https://github.com/ubuntu-core/snappy my-work
```
Comment (zyga) [inline README.md]: On second read, I would probably ask the person to fork it here.

Comment (dholbach) [inline README.md]: Yep, makes sense.

**Comment by chipaca** [comment]
:+1:

**Pull request merged**